Project 2 (13.12)/script.js

```diff
@@ -0,0 +1,95 @@
+// exercise 1
+
+// Try 1
+function reverseText(string) {
+	if (typeof string === 'string') {
+		let newString = '';
+		for (let i = string.length - 1; i >= 0; i--) {
+			newString += string[i];
+		}
+		console.log(newString);
+	} else console.log('it is not a string');
+}
+
+reverseText('piesek');
+
+//   Try 2
+function reverse() {
+	return (newString = string.split('').reverse().join(''));
```

Comment:
a gdyby go poprawnie przyjęła to po prostu `return string.split('').reverse().join(''));`